tests/e2e/upgrade/upgrade.wdio-spec.js

```diff
@@ -117,17 +114,15 @@ describe('Performing an upgrade', () => {
       state: 'finalized',
     });
 
-    if (!testFrontend) {
```

Comment:
@tatilepizs I reduced the usage of these weird IFs, at the end, these are same asserts but clearer.